charts/azuremonitor-containers/templates/ama-logs-secret.yaml

```diff
@@ -20,7 +20,7 @@ data:
   {{- else if ne .Values.amalogs.proxy "<your_proxy_config>" }}
   PROXY: {{ .Values.amalogs.proxy | b64enc | quote }}
   {{- end }}
-  {{- if and (.Values.Azure.proxySettings.isProxyEnabled) (.Values.Azure.proxySettings.proxyCert) (not .Values.amalogs.ignoreExtensionProxySettings) }}
+  {{- if and (or .Values.Azure.proxySettings.isProxyEnabled .Values.Azure.proxySettings.isCustomCert) (.Values.Azure.proxySettings.proxyCert) (not .Values.amalogs.ignoreExtensionProxySettings) }}
   PROXYCERT.crt: {{.Values.Azure.proxySettings.proxyCert | b64enc | quote}}
```

Comment:
are you making change in Arc K8s extension side to have the value of customCert to proxyCert variable?

Reply:
The change of Arc extension is ready at last October. See the note in document [here](https://learn.microsoft.com/en-us/azure/azure-arc/kubernetes/quickstart-connect-cluster?tabs=azure-cli#connect-using-an-outbound-proxy-server).

```
--custom-ca-cert is an alias for --proxy-cert. Either parameters can be used interchangeably. Passing both parameters in the same command will honour the one passed last.
```

In the case of custom CA, `.Values.Azure.proxySettings.isCustomCert` will be set to `true` and the actual custom CA will be passed by `.Values.Azure.proxySettings.proxyCert`.